logistics_planning_invoicing/models/logistics_schedule.py

```diff
@@ -0,0 +1,86 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3.0 (https://www.gnu.org/licenses/lgpl-3.0.html)
+
+from odoo import models, fields, api, _
+from odoo.exceptions import ValidationError
+
+
+class LogisticsSchedule(models.Model):
+    _inherit = 'logistics.schedule'
+
+    account_move_line_id = fields.Many2one('account.move.line', readonly=True)
+    account_move_id = fields.Many2one('account.move', related='account_move_line_id.move_id', string='Invoice')
+    is_invoiceable = fields.Boolean(compute='_compute_is_invoiceable', store=True, readonly=False)
```

Comment:
Making editable this field is not a good idea. We've talked about creating another field such `is_finished`, that could be editable when a record is in `ready` state and, at the same time, the record is not invoiceable (we'll have this case later, when link with sales or purchases are developed).
And, when `is_finished` is set to `True`, state must be set to `done` as well.

Reply:
> And, when `is_finished` is set to `True`, state must be set to `done` as well.

I still miss this part

Reply:
Here https://github.com/solvosci/slv-stock/pull/96/files#diff-d4fc0549398fbf164bb52b4b2d53bae563321fcab352f492e611a95bbb13d67cR17

Reply:
Ok, that option is better.